eth/tracers/api_blocktrace.go

```diff
@@ -357,6 +358,8 @@ func (api *API) fillBlockTrace(env *traceEnv, block *types.Block) (*types.BlockT
 	}
 
 	blockTrace := &types.BlockTrace{
+		ChainID: api.backend.ChainConfig().ChainID.Uint64(),
+		Version: fmt.Sprintf("%s-%s", params.VersionWithMeta, params.CommitHash[:8]),
```

Comment:
```suggestion
		Version: fmt.Sprintf("%s-%s", params.VersionWithMeta(), params.CommitHash()[:8]),
```

Reply:
`params.VersionWithMeta` and `params.CommitHash` are `func() string` type, so don't need to change.